web/src/components/Dialog/BaseDialog.tsx

```diff
@@ -3,26 +3,32 @@ import { createRoot } from "react-dom/client";
 import { Provider } from "react-redux";
 import { ANIMATION_DURATION } from "../../helpers/consts";
 import store from "../../store";
+import { useDialogStore } from "../../store/module/dialog";
```

Comment:
export `useDialogStore` in [index.ts](https://github.com/usememos/memos/blob/main/web/src/store/module/index.ts)

```suggestion
import { useDialogStore } from "../../store/module";
```